images/node/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM openshift/origin
```

Comment:
Needs to follow the comment pattern and style guidelines established in other images.

Also, minimize the number of layers by combining your copies, adds, and runs.